icubam/db/test_synchronizer.py

```diff
@@ -142,7 +142,9 @@ def setUp(self):
       sqla.create_engine("sqlite:///:memory:", echo=True)
     )
     self.db = store_factory.create()
-    self.csv = synchronizer.CSVSynchcronizer(self.db)
+    self.csv = synchronizer.CSVSynchcronizer(
+      self.db, 'resources/finess.geojson'
```

Comment:
should we make a `resources/fr/finess.geojson` ?

The fact that this path works in tests, is it reliant on the folder from with pytest is started. E.g. does,
```
cd icubam/db/
pytest test_synchronizer.py
```
still works?

Reply:
The problem is the json is huge, so we need this to fail gracefully for now.

Reply:
I made tests not rely on this file.